endpoints.json

```diff
@@ -0,0 +1,169 @@
+{
+  "GET /": {
+    "description": "serves up a json representation of all the available endpoints of the api"
+  },
+  "GET /sites": {
+    "description": "serves an array of all sites",
+    "queries": ["author_id", "site_id", "sort_by", "order"],
+    "exampleResponse": {
+      "sites": [
+        {
+          "siteId": 1,
+          "authorId": 1,
+          "siteName": "Durham Cathedral",
+          "siteDescription": "This is Durham Cathedral",
+          "siteImage": "https://myguideimages.s3.eu-west-2.amazonaws.com/durham_cathedral.jpg",
+          "siteAddress": "Durham DH1 3EH",
+          "latitude": 54.773678472624034,
+          "longitude": -1.5762204386383316,
+          "contactInfo": "0191 338 7178",
+          "websiteLink": "https://www.durhamcathedral.co.uk/"
+        }
+      ]
+    }
+  },
+  "POST /sites": {
+    "description": "Adds a new site into the sites database",
+    "queries": [],
+    "exampleRequest": {
+      "siteId": 27,
```

Comment:
In the post requests the siteId/tourId is automatically created so shouldn't be in the request body